parquet/src/arrow/mod.rs

```diff
@@ -281,6 +281,41 @@ impl ProjectionMask {
     pub fn leaf_included(&self, leaf_idx: usize) -> bool {
         self.mask.as_ref().map(|m| m[leaf_idx]).unwrap_or(true)
     }
+
+    /// Union two projection masks
+    ///
+    /// Example:
+    /// mask1 = [true, false, true]
+    /// mask2 = [false, true, true]
+    /// union(mask1, mask2) = [true, true, true]
```

Comment:
The rendered docs look.a bit off:
![Screenshot 2025-01-03 at 5 58 49 AM](https://github.com/user-attachments/assets/08ee9fe6-757c-4577-9f13-841e35bbe614)

I pushed a commit to make them a bit nicer

```suggestion
    /// Example:
    /// mask1 = [true, false, true]
    /// mask2 = [false, true, true]
    /// union(mask1, mask2) = [true, true, true]
```